configs/vid/temporal_roi_align/README.md

```diff
@@ -0,0 +1,28 @@
+# Temporal RoI Align for Video Object Recognition
+
+## Introduction
+
+[ALGORITHM]
+
+```latex
+@inproceedings{gong2021temporal,
+  title={Temporal ROI Align for Video Object Recognition},
+  author={Gong, Tao and Chen, Kai and Wang, Xinjiang and Chu, Qi and Zhu, Feng and Lin, Dahua and Yu, Nenghai and Feng, Huamin},
+  booktitle={Proceedings of the AAAI Conference on Artificial Intelligence},
+  volume={35},
+  number={2},
+  pages={1442--1450},
+  year={2021}
+}
+```
+
+## Results and models on ImageNet VID dataset
+
+We observe around 0.5 mAP fluctuations in performance, and provide the best model.
+
+We find that, in `SELSA + TemporalRoIAlign` method, using 3 selsa modules can bring around 0.2 mAP gains than using 2 selsa modules, while in `SELSA` method, doing so will degrade the performance by around 0.5 mAP. Therefore, we use 3 selsa modules in `SELSA + TemporalRoIAlign` method and 2 selsa modules in `SELSA` method.
```

Comment:
Note that the numbers of selsa modules in this method and `SELSA` are 3 and 2 respectively. This is because another selsa modules improve this method by 0.2 points but degrade `SELSA` by 0.5 points. We choose the best settings for the two methods for a fair comparison.